app/models/ncr/work_order.rb

```diff
@@ -13,11 +13,15 @@ def self.table_name_prefix
 
   class WorkOrder < ActiveRecord::Base
     include ObservableModel
+    include ValueHelper
+    # TODO include ProposalDelegate
 
     has_one :proposal, as: :client_data
     include ProposalDelegate
 
     after_initialize :set_defaults
+    #creates a comment containing the changes
+    before_update :changed_attributes_comment
```

Comment:
Maybe make the comment obsolete by naming the callback `:record_changes` or something?